### PR TITLE
Test md link check

### PR DESCRIPTION
**Pull request by samjwu**
testing https://github.com/RadeonOpenCompute/rocm-docs-core/pull/395

uses https://github.com/gaurav-nelson/github-action-markdown-link-check (Github action for https://github.com/tcort/markdown-link-check)

### CODE REVIEW

**Comment by samjwu** [comment]
```
ERROR: 3 dead links found!
[✖] ../tutorials/install/pytorch-install → Status: 400
[✖] ../tutorials/install/tensorflow-install → Status: 400
[✖] ../tutorials/install/magma-install → Status: 400
```
problem: cannot parse myst cross-reference links

**Comment by samjwu** [comment]
```
ERROR: 9 dead links found!
[✖] ./mi200 → Status: 400
[✖] https://www.amd.com/system/files/TechDocs/instinct-mi200-cdna2-instruction-set-architecture.pdf → Status: 0
[✖] https://www.amd.com/system/files/documents/amd-cdna2-white-paper.pdf → Status: 0
[✖] ./mi100 → Status: 400
[✖] https://www.amd.com/system/files/TechDocs/instinct-mi100-cdna1-shader-instruction-set-architecture%C2%A0.pdf → Status: 0
[✖] https://www.amd.com/system/files/documents/amd-cdna-whitepaper.pdf → Status: 0
[✖] ./w6000-v620 → Status: 400
[✖] https://www.amd.com/system/files/TechDocs/rdna2-shader-instruction-set-architecture.pdf → Status: 0
[✖] https://www.amd.com/system/files/documents/rdna2-explained-radeon-pro-W6000.pdf → Status: 0
```
problem: not clear. status code 0 can occur for various reasons, like request time out

**Comment by samjwu** [comment]
```
ERROR: 1 dead links found!
[✖] https://icl.utk.edu/magma/index.html → Status: 500
```

problem: 500 Internal Server Error. this may be due to redirects?

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -18,3 +18,13 @@ jobs:
   call-workflow-passing-data:
     name: Documentation
     uses: RadeonOpenCompute/rocm-docs-core/.github/workflows/linting.yml@develop
+
```
Comment (saadrahim) [inline .github/workflows/linting.yml:21]: this section should move to rocm-docs-core after the test period is complete.

Comment (samjwu) [reply .github/workflows/linting.yml:21]: https://github.com/RadeonOpenCompute/rocm-docs-core/pull/395

**Comment by samjwu** [comment]
[0_Markdown Link Check.txt](https://github.com/RadeonOpenCompute/ROCm/files/13290106/0_Markdown.Link.Check.txt)

**Pull request closed without merging**